notes.md

```diff
@@ -0,0 +1,4 @@
+## Testing steps
```

Comment:
Can we get rid of this file? (And if we do squash that into another commit?)

Reply:
🤦‍♂️

Reply:
😁